<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbLogin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations sur l'utilisateur</string>
   </property>
   <widget class="QLabel" name="lblMDP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUtilisateur">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom d'utilisateur:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUtilisateur">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMDP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="btnboxUtilisateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAnnuler">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbInfoServeur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations sur le serveur</string>
   </property>
   <widget class="QDialogButtonBox" name="btnboxServeur">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblIPServeur">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP du serveur:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIPServeur">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnboxUtilisateur</sender>
   <signal>accepted()</signal>
   <receiver>login</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnboxUtilisateur</sender>
   <signal>rejected()</signal>
   <receiver>login</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
